Zend/zend_ast.h

```diff
@@ -193,7 +193,7 @@ ZEND_API zend_ast *zend_ast_create(zend_ast_kind kind, ...);
 
 ZEND_API zend_ast *zend_ast_create_decl(
 	zend_ast_kind kind, uint32_t flags, uint32_t start_lineno, zend_string *doc_comment,
-	zend_string *name, zend_ast *child0, zend_ast *child1, zend_ast *child2
+	zend_string *name, zend_ast *child0, zend_ast *child1, zend_ast *child2, zend_ast *child3
```

Comment:
This is for PHP7

On Wed, Jan 28, 2015 at 5:23 AM, Andrea Faulds <notifications@github.com>
wrote:

> In Zend/zend_ast.h
> <https://github.com/php/php-src/pull/997#discussion_r23661647>:
>
> > @@ -193,7 +193,7 @@ ZEND_API zend_ast *zend_ast_create(zend_ast_kind kind, ...);
> >
> >  ZEND_API zend_ast *zend_ast_create_decl(
> >  	zend_ast_kind kind, uint32_t flags, uint32_t start_lineno, zend_string *doc_comment,
> > -	zend_string *name, zend_ast *child0, zend_ast *child1, zend_ast *child2
> > +	zend_string *name, zend_ast *child0, zend_ast *child1, zend_ast *child2, zend_ast *child3
>
>  x.y.z to x.y+1.z
>
>    - Bugfixes
>    - New features
>    - Extensions support can be ended (moved to pecl)
>    - Backward compatibility must be kept
>    - API compatibility must be kept (userland)
>    - ABI and API can be broken (internals), src compatibility should be
>    kept if possible, while breakages are allowed
>
>  —
> Reply to this email directly or view it on GitHub
> <https://github.com/php/php-src/pull/997/files#r23661647>.
>